profiles/api/serializers.py

```diff
@@ -4,21 +4,21 @@
 from django.contrib.auth.models import User
 from rest_framework import serializers
 
-from ..models import PlayerCharacter, DMNote, Profile
+from ..models import PlayerCharacter, Profile
 
 logger = logging.getLogger(__name__)
 
 
 class PublicPlayerCharacterSerializer(serializers.ModelSerializer):
     class Meta:
         model = PlayerCharacter
-        fields = ("id", "name", "pc_class", "race", "level", "faction", "notes")
+        fields = ("id", "name", "pc_class", "race", "level", "faction", "notes","dead")
```

Comment:
There should be space after comma. This is not exlicitly stated in PEP8, but most codestyle checkers will mark it as E251.